nginx-conf/nginx.conf

```diff
@@ -113,6 +113,9 @@ http {
             fastcgi_intercept_errors on;
             # changed from drupal-fpm to localhost for azure containers
             fastcgi_pass localhost:9000;
+            # make drupal understand clients were talking https to the loadbalancer
+            fastcgi_param HTTPS on;
```

Comment:
I think these should be moved up a few lines so they're with the other fastcgi_params